packages/typespec-vscode/test/extension/common/utils.ts

```diff
@@ -0,0 +1,144 @@
+import { execSync } from "child_process";
+import fs from "node:fs";
+import os from "node:os";
+import path, { resolve } from "node:path";
+import { Page, _electron } from "playwright";
+import { test as baseTest, inject } from "vitest";
+import { closeVscode } from "./common-steps";
+
+const __dirname = import.meta.dirname;
+const projectRoot = path.resolve(__dirname, "../../../");
+const imagesPath = path.resolve(projectRoot, "test/extension/images-linux");
```

Comment:
Keep those in `temp` folder too, rioght now this create files that are not git ignored